app/utils/wallet/chainConfig.tsx

```diff
@@ -60,6 +60,7 @@ const singleWalletChains = [
   {
     id: "dogecoin",
     name: "Dogecoin",
+    thorchainIdentifier: "doge",
```

Comment:
Why can't we use id here?

Reply:
We need a way to match the short name from thorchain to our configs, "eth, bsc, doge, btc".